lib/src/Pulsar/Internal/Core.hs

```diff
@@ -50,7 +50,7 @@ runPulsar' (LogOptions lvl out) (Connection mgd) (Pulsar mr) = do
   finalizers ctx = do
     let (worker, connVar) = ctxConnWorker ctx
     app <- readIORef (ctxState ctx)
-    traverse_ (\(a, v) -> putMVar v () >> cancel a) (appWorkers app)
+    traverse_ (\(a, v) -> putMVar v () >> cancel a) (_appWorkers app)
```

Comment:
`traverseOf_` ?

Reply:
Didn't find something like that, unfortunately :| http://hackage.haskell.org/package/lens-family-core-2.1.0/docs/Lens-Family-Stock.html#g:5